mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/bridge/IdentifierBridge.java

```diff
@@ -61,6 +61,8 @@ default void bind(IdentifierBridgeBindingContext<I> context) {
 	 */
 	I fromDocumentIdentifier(String documentIdentifier, IdentifierBridgeFromDocumentIdentifierContext context);
 
+	boolean isCompatibleWith(IdentifierBridge<?> bridge);
```

Comment:
We use `equals( other )` as a default implementation for value bridges; I think it would be appropriate here, too.